File: removed/CPP/fizz-buzz.cpp
#include <iostream>
using std::cout;

int main()
{
	for (int i = 1; i <= 10000; i++)
	{
		if (i % 15 == 0)
		{
			cout << "FizzBuzz\n";
		}
		else if(i % 3 == 0)
		{
			cout << "Fizz\n";
		}
		else if(i % 5 == 0)
		{
			cout << "Buzz\n";
		}
		else
		{
			cout << i << "\n";
		}
	}
}

File: template/Ruby/fizz-buzz.rb
#!/usr/bin/env ruby

(1 .. FIZZBUZZ_CYCLES).each do |i|
    divides = lambda { |mod| return (i % mod == 0); }
    div3 = divides.call(3)
    div5 = divides.call(5)

    puts ((div3 && div5) ? "FizzBuzz" :
        div3 ? "Fizz" :
        div5 ? "Buzz" :
        i
    )
end

__END__